rules/aip0162/rollback_response_message_name.go

```diff
@@ -30,6 +32,15 @@ var rollbackResponseMessageName = &lint.MethodRule{
 		// message is `Book`.
 		want := rollbackMethodRegexp.FindStringSubmatch(m.GetName())[1]
 		got := m.GetOutputType().GetName()
+		loc := locations.MethodResponseType(m)
+
+		// If LRO, check the response_type short name.
+		if utils.IsOperation(m.GetOutputType()) {
```

Comment:
I think it could be useful for sure.

I think at least _some_ of the code that does similar checks to extract the response type will also want to know if it is an LRO or not for another reason (i.e. which `Location` to provide to the finding).

Could be a nice clean up though.